tests/unit/converters/test_line_of_sight.py

```diff
@@ -2,193 +2,136 @@
 
```

Comment:
Some functions are missing from these unit tests: `convert_from_Rz`, `convert_to_rho`(though this is already noted in the code), `map_to_los`, `check_rho_and_profile`, `integrate_on_los` and `calc_impact_parameter`. I don't know if any unit test for `plot_los` or `example_run` is necessary but I leave that up to you.